.github/workflows/cursor-bot-pr-review-agent.yml

```
@@ -0,0 +1,261 @@
+          else
+            echo "no_comments=false" >> "$GITHUB_OUTPUT"
+          fi
+
+      - name: Exit if no bot comments
+        if: steps.checks.outputs.missing_required == 'false' && steps.idempotency.outputs.already_launched != 'true' && steps.comments.outputs.no_comments == 'true'
+        run: echo "No unresolved bot comments; skipping agent launch."
+
+      - name: Build bot review context and prompt
+        id: prompt
+        if: steps.checks.outputs.missing_required == 'false' && steps.idempotency.outputs.already_launched != 'true' && steps.comments.outputs.no_comments != 'true'
+        run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2034:warning:2:1: HEAD_REF appears unused. Verify use (or export if used externally)

(shellcheck)